fabfile/utils.py

```diff
@@ -194,3 +195,28 @@ def raw_input_wrapper(query, lower=False):
     if lower:
         usr_input = usr_input.lower()
     return usr_input
+
+
+def pretty_print_dictionary(dictionary):
```

Comment:
would just using python's built in one be good enough? https://docs.python.org/2/library/pprint.html